packages/react-router-redux/modules/ConnectedRouter.js

```diff
@@ -26,7 +26,9 @@ class ConnectedRouter extends Component {
               payload: location
             })
 
-            return children
+            return !Array.isArray(children) || children.length ? ( // Preact defaults to empty children array
+              React.Children.only(children)
+            ) : null
```

Comment:
Is it okay to put the `invariant` check inside `render`? Or would you rather see it inside `componentWillMount` like it is in `<Router>`?